packages/components/src/Actions/Action/Action.component.js

```diff
@@ -88,6 +89,10 @@ function Action(props) {
 		model,
 	});
 
+	if (form) {
+		buttonProps.form = form;
+	}
```

Comment:
I don't think you need that if statement. The property could be passed directly as a prop to Button.